lib/traces/backend/open_telemetry/interface.rb

```diff
@@ -33,7 +33,7 @@ module OpenTelemetry
 			TRACER = ::OpenTelemetry.tracer_provider.tracer(Traces::Backend::OpenTelemetry.name, Traces::Backend::OpenTelemetry::VERSION)
 			
 			module Interface
-				def trace(name, attributes: nil, &block)
+				def trace(name, attributes: {}, &block)
```

Comment:
`attributes` calls `transform_keys` below, which `nil` will not respond to.

Reply:
Yes, that makes sense, alternatively we could write `attributes&.transform_keys`. Not sure what makes more sense.